examples/processors/notify_on_failure.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module SolidusImporter
+  module Processors
+    ##
+    # This processor sends an email if the process of a single row is failing.
+    #
+    # Install: add the class to processors in solidus_importer configuration
+    class NotifyOnFailure < Base
+      def call(context)
+        return if context[:success]
```

Comment:
shouldn't this return the original context here?

Reply:
No, actually processors only updates received context.